priv/repo/migrations/20241023111912_alter_operator_sign_ins_radio.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Orbit.Repo.Migrations.AlterSigninsRadio do
+  use Ecto.Migration
+
+  def change do
+    alter table(:operator_sign_ins) do
+      add :radio_number, :integer
```

Comment:
Are radio numbers actually integers, or should they be strings (made of digits), like badge numbers / zip codes?

Reply:
Yeah, I agree 👍 changed